package.json

```diff
@@ -94,6 +96,49 @@
     "ci": "concurrently \"npm run build\" \"npm test\"",
     "package-plugin": "./bin/build-plugin-zip.sh",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "test-unit:jest": "jest",
+    "test-unit:coverage": "jest --coverage",
+    "test-unit:watch": "jest --watch"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "**/blocks/**/*.js",
+      "**/components/**/*.js",
+      "**/date/**/*.js",
+      "**/editor/**/*.js",
+      "**/element/**/*.js",
+      "**/i18n/**/*.js",
+      "**/utils/**/*.js"
+    ],
+    "coveragePathIgnorePatterns": [
+      "<rootDir>/components/clipboard-button/index.js",
```

Comment:
It fails because nyc or istanbul doesn't like param destructuring in function definition that uses default value ...

I think it needs to be fixed somewhere else. I can investigate it further later.